src/Microsoft.Framework.Runtime/Servicing/Breadcrumbs.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.Framework.Runtime.Servicing
     public class Breadcrumbs
     {
         // It is recommended that you disable breadcrumbs only for perf runs
-        private const string NoBreadcrumbsEnvironmentVariableName = "DNX_NO_BREADCRUMBS";
+        private const string NoBreadcrumbsEnvironmentVariableName = Constants.RuntimeShortName + "_NO_BREADCRUMBS";
```

Comment:
I think this it will be forever, and that's a bad reason to have inconsistent casing. Please make it uppercase DNX_